Extensions/Multiplayer/messageManager.ts

```diff
@@ -1,5 +1,12 @@
 namespace gdjs {
   const logger = new gdjs.Logger('Multiplayer');
+  /* Set to true to see message logs and ease debugging. */
+  export const ENABLE_DEBUG_LOGS = false;
```

Comment:
Should it be a variable that can be set with an action to help users debug their multiplayer games?

Reply:
I like the idea, though I think it's too low level for anyone to be able to understand what they mean and help debug their game.